### PR TITLE
alert -> error to danger

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

https://gitcoincore.slack.com/archives/G01LRE6F39A/p1617025362000900

### CODE REVIEW

File: app/assets/v2/js/board.js [hunk 1]
```diff
@@ -435,7 +435,7 @@ Vue.mixin({
 
               await updatePtokenStatusinDatabase(transactionHash, successMsg, errorMsg);
             })
-            .on('error', function(err) {
+            .on('danger', function(err) {
```
Comment (gdixon) [inline app/assets/v2/js/board.js:438]: These need changing back to `.on('error')`

File: app/assets/v2/js/board.js [hunk 2]
```diff
@@ -459,7 +459,7 @@ Vue.mixin({
                 const errorMsg = 'Oops, something went wrong increasing your token supply. Please try again or contact support@gitcoin.co';
 
                 await updatePtokenStatusinDatabase(transactionHash, successMsg, errorMsg);
-              }).on('error', function(err) {
+              }).on('danger', function(err) {
```
Comment (gdixon) [inline app/assets/v2/js/board.js:462]: `.on('error')`

File: app/assets/v2/js/board.js [hunk 3]
```diff
@@ -479,7 +479,7 @@ Vue.mixin({
                 const errorMsg = 'Oops, something went wrong decreased your token supply. Please try again or contact support@gitcoin.co';
 
                 await updatePtokenStatusinDatabase(transactionHash, successMsg, errorMsg);
-              }).on('error', function(err) {
+              }).on('danger', function(err) {
```
Comment (gdixon) [inline app/assets/v2/js/board.js:482]: `.on('error')`

File: app/assets/v2/js/board.js [hunk 4]
```diff
@@ -545,7 +545,7 @@ Vue.mixin({
         const errorMsg = 'Oops, something went wrong trying to create your time token. Please try again or contact support@gitcoin.co';
 
         await updatePtokenStatusinDatabase(transactionHash, successMsg, errorMsg);
-      }).on('error', function(err) {
+      }).on('danger', function(err) {
```
Comment (gdixon) [inline app/assets/v2/js/board.js:548]: `.on('error')`

File: app/assets/v2/js/board.js [hunk 5]
```diff
@@ -623,7 +623,7 @@ Vue.mixin({
 
             await updatePtokenStatusinDatabase(transactionHash, successMsg, errorMsg);
             vm.checkData('personal-tokens');
-          }).on('error', async(error, receipt) => {
+          }).on('danger', async(error, receipt) => {
```
Comment (gdixon) [inline app/assets/v2/js/board.js:626]: `.on('error')`

**Review by gdixon — CHANGES_REQUESTED** [review]
A couple of `.on('error')` events got caught up in the replacement

**Comment by thelostone-mc** [comment]
@gdixon updated 🙌

**Review by gdixon — APPROVED** [review]

**Pull request merged**